Laboratoare/Lab/src/polimorfism/birdcontrollerex1/hierarchyex5/model/BikingShoes.java

```diff
@@ -0,0 +1,20 @@
+package polimorfism.birdcontrollerex1.hierarchyex5.model;
+
+
+public class BikingShoes extends SportShoes {
+
+    private int rotationNo = 0;

```

Comment:
nu initializa niciodata un field aici. Initializeaza-l in constructor.
Poti citi mai multe despre ordinea in care sunt initializate chestiile in Java aici si vei intelege de ce spun ca nu e bine sa initializez aici un field, ci e mai bine in constructor: http://stackoverflow.com/questions/23093470/java-order-of-initialization-and-instantiation